requirements.txt

```diff
@@ -1,6 +1,6 @@
-YTClient
 celery[redis]
 future
 pyyaml
 montagu>=0.0.2
 orderlyweb-api>=0.0.10
+git+https://github.com/reside-ic/youtrack-rest-python-library
```

Comment:
I had to fork the limited YT client we were using to allow updating tickets and fetching/creating tags https://github.com/reside-ic/youtrack-rest-python-library